funcbench/README.md

```diff
@@ -24,10 +24,21 @@ funcbench currently supports two modes, Local and GitHub. Running it in the Gith
 ## Triggering with GitHub comments
 
 The benchmark can be triggered by creating a comment in a PR which specifies a branch to compare. The results are then posted back to the PR as a comment.
+
+The syntax is: `/funcbench <branch/commit> <benchmark function regex>`
 ```
-/funcbench <branch/commit> <benchmark function regex>
+# test all the BenchmarkQuery.* on branch master vs this pull request you commented on
+/funcbench master BenchmarkQuery.*
+
+# test all the bencmarks on feature branch vs this pull request
+/funcbench feature-branch
+
+or
+
+/funcbench feature-branch . *
```

Comment:
I think it'll be nice if we format the comment instructions like this:
https://github.com/prometheus/test-infra/tree/master/prombench#trigger-tests-via-a-github-comment

It's more clear. Also I think `/funcbench feature-branch . * ` should be `
/funcbench feature-branch .*` without the space between `.` and `*`

Reply:
Oh, the space is a mistake :P